test/introspection/SupportsInterface.behavior.js

```diff
@@ -14,38 +14,59 @@ const INTERFACE_IDS = {
     'transferFrom(address,address,uint256)',
     'safeTransferFrom(address,address,uint256)',
     'safeTransferFrom(address,address,uint256,bytes)',
-  ]),
-  ERC721Enumerable: makeInterfaceId([
+  ],
+  ERC721Enumerable: [
     'totalSupply()',
     'tokenOfOwnerByIndex(address,uint256)',
     'tokenByIndex(uint256)',
-  ]),
-  ERC721Metadata: makeInterfaceId([
+  ],
+  ERC721Metadata: [
     'name()',
     'symbol()',
     'tokenURI(uint256)',
-  ]),
-  ERC721Exists: makeInterfaceId([
+  ],
+  ERC721Exists: [
```

Comment:
Is this still in use?

Reply:
You're right - that is an obsolete interface. I've removed it and updated the PR